### ISSUE TITLE
Review google search indexing configuration to ensure `noindex` implemented correctly for v1 docs ie https://docs-legacy.prefect.io/

### ISSUE THREAD

**Issue by thomas-te** [PrefectHQ/prefect#14467]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used GitHub search to find a similar request and didn't find it 😇

### Describe the issue

Users of Prefect 2 or 3 may unintentionally find a google search result from legacy docs. This is confusing.

### Describe the proposed change

Ensure legacy docs site is not indexed by google or other search engines. Consider adding very subtle link out to legacy docs from docs.prefect.io as per @billpalombi suggestion

### Additional context

For visibility @sa

**Comment by znicholasbrown** [comment]
Unfortunately I'm not sure we'll be able to make this happen for that particular site without taking on a pretty substantial refactor of the build to move to modern systems. That site was previously built with node 10 and python 3.7, neither of which are supported by netlify's build image any longer. I attempted to update the build locally but would need to replace the `node-sass` package used by vuetify because it runs off python 2(!!!!) and was generally throwing a LOT of errors on python 3.8. 

I've merged the [code change](https://github.com/PrefectHQ/legacy-api-docs/pull/50) that would disable search indexing should we want to take on the challenge of rebuilding the site but my recommendation would be that we actually de-list the site since its last supported version was published [over 4 years ago](https://github.com/PrefectHQ/prefect/releases/tag/0.12.6).
